### PR TITLE
usersession: support additional zoom URL schemes

### PR DESCRIPTION
**Pull request by zyga**
We've recently added support for the zoommtg:// URL scheme
but zoom documentation also mentions two others, zoomus and
zoomphonecall.

Ref: https://marketplace.zoom.us/docs/guides/guides/client-url-schemes
Fixes: https://bugs.launchpad.net/snapd/+bug/1880606
Signed-off-by: Zygmunt Krynicki <me@zygoon.pl>

### CODE REVIEW

File: usersession/userd/launcher.go [hunk 1]
```diff
@@ -104,7 +104,7 @@ const launcherIntrospectionXML = `
 //   - https://github.com/snapcore/snapd/pull/8304
 
 var (
-	allowedURLSchemes = []string{"http", "https", "mailto", "snap", "help", "apt", "zoommtg", "slack", "msteams"}
+	allowedURLSchemes = []string{"http", "https", "mailto", "snap", "help", "apt", "zoommtg", "zoomus", "zoomphonecall", "slack", "msteams"}
```
Comment (bboozzoo) [inline usersession/userd/launcher.go:107]: Can we format the list vertically?

Comment (bboozzoo) [inline usersession/userd/launcher.go:107]: I'm a bit confused looking at the documentation. Isn't `zoomus://` only used on ios/android?

Comment (zyga) [reply usersession/userd/launcher.go:107]: Sure!

Comment (zyga) [reply usersession/userd/launcher.go:107]: It's provided by mobile clients but you can paste the URL and open it on a computer. I really don't understand why they have two URL schemes for the same thing.

**Pull request closed without merging**